.infrastructure/cronjob.yml

```diff
@@ -0,0 +1,30 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: todoapp-cronjob
+  namespace: mateapp
+spec:
+  schedule: "*/4 * * * *"
+  concurrencyPolicy: Allow
```

Comment:
The `concurrencyPolicy` should be `Allow` instead of `Allow`. This is case-sensitive and should be corrected.